site/source/docs/tools_reference/settings_reference.rst

```diff
@@ -151,6 +151,17 @@ Note that this setting does not affect the behavior of operator new in C++.
 This function will always abort on allocation failure if exceptions are disabled.
 If you want new to return 0 on failure, use it with std::nothrow.
 
+.. _initial_heap:
+
+INITIAL_HEAP
+============
+
+The initial amount of heap memory available to the program.  This is the
+memory region available for dynamic allocations via `sbrk`, `malloc` and `new`.
+
+Unlike INITIAL_MEMORY, this setting does not place a hard cap on the total
+amount of memory available at startup, and so should be preferred.
```

Comment:
I'm not sure I love in the sentence.

How about: 

```
Unlike INITIAL_MEMORY, this setting allows the static and dynamic regions of your programs memory to independently grow.  In most cases we recommend using this setting rather than `INITIAL_MEMORY`.  However, this setting does not work for imported memories (e.g. when dynamic linking is used).
```